classes/class-wc-connect-shipping-method.php

```diff
@@ -438,5 +470,19 @@ public static function format_rate_title( $rate_title ) {
 			return $formatted_title;
 		}
 
+		/**
+		 * Log debug by printing it as notice.
+		 *
+		 * @param string $message Debug message.
+		 * @param string $type    Notice type.
+		 */
+		public function debug( $message, $type = 'notice' ) {
+			if ( is_cart() || is_checkout() ) {
```

Comment:
We probably don't want to show anything when updating the cart from other pages, but what about upon "Update cart"? Not totally sure how we'd support it since this check seems to exclude it.

Reply:
I think https://github.com/Automattic/woocommerce-services/pull/1303/commits/4f07b2323da4a944f3ee6f1a81f18c6c686ccb2c should do it - only the cart update handler uses `update_cart` in the post body.